documentation/applying_manifest_blocks.md

```diff
@@ -223,6 +223,55 @@ $results.each |$result| {
 }
 ```
 
+### Report keys
+
+The [`ApplyResult`
+object](https://puppet.com/docs/bolt/latest/bolt_types_reference.html#applyresult)
+includes a `report` method that returns a hash representation of the
+[`Puppet::Transaction::Report`](https://puppet.com/docs/puppet/7.3/format_report.html)
+object. Each property on the object corresponds to a key with the same name in
```

Comment:
I think it's worth noting that not every property will be represented in the hash for every result, I believe it's only those that have values from the apply (for example I don't think `job_id` will ever be in the report in Bolt)